src/test/java/htsjdk/samtools/util/Md5CalculatingOutputStreamTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2014 The Broad Institute
```

Comment:
Hm - thats a bit out of date.

Reply:
I copy&pasted this from elsewhere in the project, can find a newer one.

Reply:
Ha, so the issue with headers is a bit more severe - I saw several somewhat different styles in files. So I simply bumped the year to 2019.